software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/couchdb/CouchDBNodeSshDriver.java

```diff
@@ -198,24 +211,36 @@ public void customize() {
     @Override
     public void launch() {
         log.info("Launching  {}", entity);
-        newScript(MutableMap.of(USE_PID_FILE, false), LAUNCHING)
-                .body.append(String.format("nohup ./bin/couchdb -p %s -a %s -o couchdb-console.log -e couchdb-error.log -b &", getPidFile(), Os.mergePathsUnix(getRunDir(), getCouchDBConfigFileName())))
-                .execute();
+        String couchDBPath = isV2()? getExpandedInstallDir()+"/rel/couchdb/bin/couchdb" : "./bin/couchdb";
+        ScriptHelper script = newScript(MutableMap.of(USE_PID_FILE, false), LAUNCHING);
```

Comment:
`USE_PID_FILE` would write it to a `pid.txt` whereas this expects a `couchdb.pid`. I could swap it but I think it might break the 1.x version.